.npmignore

```diff
@@ -0,0 +1,12 @@
+!dist/
```

Comment:
Yes, this is where the compiled code is which needs to be distributed.

`npm pack` first looks at your `.gitignore`, then at your `.npmignore` to see which files it should tar up into the npm package it will upload.

Since `dist/` is in `.gitignore`, we need to explicitly unignore it so that it is packed into the tarball.